system-int-test/src/cmr/system_int_test/utils/subscription_util.clj

```diff
@@ -7,21 +7,24 @@
    [cmr.common.mime-types :as mime-types]
    [cmr.mock-echo.client.echo-util :as echo-util]
    [cmr.system-int-test.data2.core :as d]
+   [cmr.system-int-test.data2.granule :as dg]
    [cmr.system-int-test.data2.umm-spec-subscription :as data-umm-sub]
    [cmr.system-int-test.system :as s]
    [cmr.system-int-test.utils.index-util :as index]
    [cmr.system-int-test.utils.ingest-util :as ingest-util]
+   [cmr.system-int-test.utils.metadata-db-util :as mdb]
    [cmr.system-int-test.utils.search-util :as search]
    [cmr.system-int-test.utils.url-helper :as urls]
    [cmr.transmit.config :as config]
    [cmr.transmit.connection :as conn]
    [cmr.transmit.search :as transmit-search]
+   [cmr.umm.echo10.echo10-core :as echo10]
    [cmr.umm-spec.versioning :as versioning]))
 
 (def versioned-content-type
   "A versioned default content type used in the tests."
   (mime-types/with-version
-   mime-types/umm-json versioning/current-subscription-version))
+    mime-types/umm-json versioning/current-subscription-version))
```

Comment:
It might just be me, but the calva linter seems inconsistent... why did this get an extra whitespace?

Reply:
Eagle eye!  Calva cannot interpret mime-types/umm-json because it's added to the namespace dynamically, thus failing when formatting.

Space manually adjusted.